src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,8 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+import mate.academy.model.User;
+
+public class AuthenticationService extends UserService {
```

Comment:
UserService should be class field

```suggestion
public class AuthenticationService {
```

Reply:
I do not understen how i can fix this. What is meen UserService should be class field